Source/WebCore/dom/Element.cpp

```diff
@@ -1167,8 +1167,6 @@ void Element::scrollIntoView(std::optional<std::variant<bool, ScrollIntoViewOpti
 
 void Element::scrollIntoView(bool alignToTop) 
 {
-    protectedDocument()->updateLayoutIgnorePendingStylesheets(LayoutOptions::UpdateCompositingLayers);
-
```

Comment:
is this what you wanted to remove? btw, figuring out the computed style still needs some work, what I meant that it does not require the layout part (i.e. there has to be an API on Document which takes care of resolving the styles but does not run layout)

Reply:
Ops! Wrong change.

Reply:
I wanted to do in `currentCSSZoom` and also `localZoomForRenderer` looks interesting, is this what you meant?

Reply:
NOTE - without layout, we will fail WPT as can be seen from last run.

Reply:
Could you check again?  did you omit only the _layout_ call or both style resolve and layout?

Reply:
that's what I was trying to imply with my comments above. i.e. find a function on Document that calls style update but skips layout (as for this API we don't need to have uptodate geometry, only computed style).